samples/Azure.Management.Storage/Generated/BlobServicePropertiesContainer.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Why did this become BlobServicePropertiesContainer instead of BlobServiceContainer?

Reply:
Because the ResourceName for the operation "BlobService" is "BlobServiceProperties": https://github.com/Azure/azure-rest-api-specs/blob/e606243e5297312781dd7dbfd7ab76d2329cc088/specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/blob.json#L91